test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageArchiveReaderTests.cs

```diff
@@ -1868,7 +1868,32 @@ private static byte[] CreateNonEmptyZipArchive()
         }
 #endif
 
-        private static string ExtractFile(string sourcePath, string targetPath, Stream sourceStream)
+
+        [Fact]
+        public void CanVerifySignedPackages_Platform()
```

Comment:
The `Platform` suffix doesn't explain very well what the test is doing.  A better test name would be:

    CanVerifySignedPackages_ReturnsValueBasedOnOperatingSystemAndFramework